dashboard/src/components/myProgress/MyProjectSkillsPage.vue

```diff
@@ -14,17 +14,24 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 <template>
-<div>
-  <div v-if="isLoadingSettings" class="d-flex justify-content-center mt-1">
-    <b-spinner variant="primary" type="grow" label="Spinning"></b-spinner>
+  <div>
+    <div v-if="isLoadingSettings" class="d-flex justify-content-center mt-1">
+      <b-spinner variant="primary" type="grow" label="Spinning"></b-spinner>
+    </div>
+    <div class="w-100 text-right pr-3 pt-3" v-if="emailEnabled">
```

Comment:
do not show contact button until settings are loaded; add ``!isLoadingSettings`` to the ``v-if``